client/components/Widget.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+// import React from 'react-dom';
+
+class Widget extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      listingId: this.props.listingData.listingId
+    };
+  }
+
+  handleClick() {
```

Comment:
Whats the point of this handleClick function defintion? I see it gets used in your function, just a little unsure how it comes into play....

Reply:
it's not doing anything yet!  it'll come into play when i start implementing a scroll-to parts of the page and may use a modal if I have time.  just haven't built it out yet.